platform-api/che-core-api-project/src/main/java/org/eclipse/che/api/project/server/ProjectManager.java

```diff
@@ -63,6 +64,29 @@
     Project getProject(String workspace, String projectPath) throws ForbiddenException, ServerException, NotFoundException;
 
     /**
+     * Configure project after import. If project wasn't found in workspace it will be configured as blank.
+     *
+     * @param storage
+     *         storage from which project was imported
+     * @param workspaceId
+     *         workspace id
+     * @throws ServerException
+     *         if other error occurs
+     * @throws IOException
```

Comment:
Add [description](https://github.com/codenvy/codenvy-dev-resources/wiki/Formatting#generally) because `@param`, `@return`, `@throws` or `@deprecated` must not be appear without description